layouts/generic-py-base.html

```diff
@@ -31,10 +31,8 @@
         </div>
       </div>
       {% block global_subheader_container %}
-        <div class="global-subheader">
-          <div class="container">
-            {% block global_subheader %}{% endblock %}
-          </div>
+        <div class="global-subheader container">
```

Comment:
I think the idea of two `div`s here is to add flexibility of spanning the whole width of the viewport for `.global-subheader`. For stylistic purposes, e.g. when using a background or border.